packages/mobile/src/apollo/types.ts

```diff
@@ -1,41 +1,51 @@
 import { Query } from 'react-apollo'
-import { TransactionTypes } from 'src/transactions/reducer'
+import { CURRENCY_ENUM } from 'src/geth/consts'
```

Comment:
I just tried it and it doesn't work right now, but i think this file originally came from running `yarn run build:gen-graphql-types` with the blockchain api running locally

Reply:
Kk are you saying we should avoid modifying this generated file?

Reply:
This file *was* originally made that way but most of that auto gen code has been changed over time. I've made modifications here too.

Reply:
I believe we diverged away from the auto gen types because of the differing shapes of our standby tx and graphql txs